ietf/templates/meeting/requests.html

```diff
@@ -79,16 +80,12 @@ <h2 class="mt-5" id="{{ area.grouper.acronym }}">
                                             {% with constraint.source.parent.id as constraint_source_parent_id %}
                                                 {% with constraint.person as constraint_person %}
                                                     {% if constraint_target_parent_id == constraint_source_parent_id and not constraint_person %}<b>{% endif %}
-                                                    {% if constraint.name.slug == "bethere" %}
-                                                        {% person_link constraint_person %}
+                                                    {% if constraint.name.slug == "bethere" %}{% person_link constraint_person %}
                                                     {% else %}
                                                         {% with constraint.name.slug as constraint_name_slug %}
-                                                            <span class="{% if constraint_name_slug == 'conflict' %}text-danger{% elif constraint_name_slug == 'conflic2' %}text-warning{% elif constraint_name_slug == 'conflic3' %}text-success{% else %}{{ constraint_name_slug }}{% endif %}">
+                                                            <span class="{% if constraint_name_slug == 'chair_conflict' %}text-danger{% elif constraint_name_slug == 'tech_overlap' %}text-info{% elif constraint_name_slug == 'key_participant' %}text-success{% else %}{{ constraint_name_slug }}{% endif %}">
```

Comment:
This should become aware of the constraints in use by a given meeting: https://github.com/ietf-tools/datatracker/blob/main/ietf/meeting/models.py#L125.

Reply:
I can do a filter, but the line Robert indicated doesn't give me enough information about what the desired behavior is?

Reply:
this is probably bigger than this first pass and should be split into a separate ticket. A meeting knows which constraint names are in use. Right now, we have hardcoded how to present each constraint into the template (chair_conflict is text-danger, etc). We would need to add a filter, as jennifer is suggesting, that encapsulated what class to provide for each constraint type.